x-pack/plugins/ml/public/url_generator.ts

```diff
@@ -24,11 +26,81 @@ declare module '../../../../src/plugins/share/public' {
 
 export const ML_APP_URL_GENERATOR = 'ML_APP_URL_GENERATOR';
 
+export const ML_PAGES = {
+  ANOMALY_DETECTION: 'jobs',
+  ANOMALY_EXPLORER: 'explorer',
+  TIME_SERIES_EXPLORER: 'timeseriesexplorer',
+  DATA_FRAME_ANALYTICS: 'data_frame_analytics',
```

Comment:
Maybe change to `DATA_FRAME_ANALYTICS_JOBS_MANAGE`?

Reply:
Updated here https://github.com/elastic/kibana/pull/75696/commits/18b909396dd9bddc539f2dad95c481bf2462d8eb